vendor/github.com/openshift/cluster-api/pkg/controller/machine/controller.go

```diff
@@ -37,7 +45,12 @@ import (
 	"sigs.k8s.io/controller-runtime/pkg/source"
 )
 
-const NodeNameEnvVar = "NODE_NAME"
+const (
+	NodeNameEnvVar = "NODE_NAME"
+
+	// ExcludeNodeDrainingAnnotation annotation explicitly skips node draining if set
+	ExcludeNodeDrainingAnnotation = "machine.openshift.io/exclude-node-draining"
```

Comment:
there was some discussion with @frobware and @kalexand-rh on slack about reconsidering this annotation name, otherwise looks good to me

Reply:
Definitely not a blocker for the PR as we already have it merged. Though, I am fine changing the name to something more proper.